stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -63,11 +64,14 @@
     private static final String LOGGING_ENDPOINT = "https://m.stripe.com/4";
 
     private static final String CHARSET = "UTF-8";
+    private static final String CUSTOMERS = "customers";
     private static final String TOKENS = "tokens";
     private static final String SOURCES = "sources";
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
+    static final String API_VERSION = "2017-06-05";
```

Comment:
Ok. I'll see about putting in an api version dynamically as well - we should also probably start pushing up locales so that we get localized error messages (have a bug out on that one)